Assets/ScriptableObjects/Plugin Scripts/MoveablePlugin.cs

```diff
@@ -33,13 +33,23 @@ public override void OnFaceClick(BlockFace face)
         }
     }
 
+    public BlockFace GetDisplacedFace()
+    {
+        return _displacedFace;
+    }
+
+    public bool CanMove()
+    {
+        return true;
+    }
+

```

Comment:
This doesnt seem very accurate as a block may not be able to be moved when it is translating.  But see my comments in `PlayerMouse` first.